src/Features/CSharp/Portable/Completion/KeywordRecommenders/LoadKeywordRecommender.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal class LoadKeywordRecommender : AbstractSyntacticSingleKeywordRecommender
+    {
+        public LoadKeywordRecommender()
+            : base(SyntaxKind.LoadKeyword, isValidInPreprocessorContext: true)
+        {
+        }
+
+        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
+        {
+            var syntaxTree = context.SyntaxTree;
+            return
+                syntaxTree.IsInteractiveOrScript() &&
+                syntaxTree.IsBeforeFirstToken(position, cancellationToken) &&
+                context.IsPreProcessorKeywordContext;
```

Comment:
I've gone ahead and moved the ```context.IsPreProcessorKeywordContext``` check to the start of ```IsValidContext``` in all the recommenders where it would be more optimal to do so.